test/src/testauth.py

```diff
@@ -0,0 +1,10 @@
+def testauth_none():
```

Comment:
What is this file doing?

Reply:
It's meant to implement dummy functions corresponding to the actionsmap in test/actionsmap/moulitest.yml used for "unit" / functional tests